test-utils/test-runner/README.md

```diff
@@ -0,0 +1,199 @@
+# Substrate Test Runner
+
+Allows you to test
+<br />
+
+-   Migrations
+-   Runtime Upgrades
+-   Pallets and general runtime functionality.
+
+This works by running a full node with a ManualSeal-BABE™ hybrid consensus for block authoring.
+
+The test runner provides two apis of note
+
+-   `seal_blocks(count: u32)`
+    <br/>
+
+    This tells manual seal authorship task running on the node to author `count` number of blocks, including any transactions in the transaction pool in those blocks.
+
+-   `submit_extrinsic<T: frame_system::Config>(call: Impl Into<T::Call>, from: T::AccountId)`
+    <br/>
+
+    Providing a `Call` and an `AccountId`, creates an `UncheckedExtrinsic` with an empty signature and sends to the node to be included in future block.
+
+<h2>Note</h2>
+The running node has no signature verification, which allows us author extrinsics for any account on chain.
+    <br/>
+    <br/>
+
+<h2>How do I Use this?</h2>
+
+
+```rust
+/// tons of ignored imports
+use substrate_test_runner::{TestRequirements, Node};
+
+struct Requirements;
+
+impl TestRequirements for Requirements {
```

Comment:
```suggestion
impl TestRequirements for Requirements {
```

This is still outdated + for the exact same reason, you should remove the MD file and put this in the module doc so it is compiled, and you remember to update it once you break things :P